sdk/tests/tutorials/marketdata/test_corporate_actions.py

```diff
@@ -0,0 +1,30 @@
+import unittest
+
+import lusid
+import lusid.models
+from lusid.utilities.api_client_builder import ApiClientBuilder
+from lusid.exceptions import ApiException
+from utilities.credentials_source import CredentialsSource
+from utilities.test_data_utilities import TestDataUtilities
+
+
+class CorporateActions(unittest.TestCase):
+
+    @classmethod
+    def setUp(cls):
+        # create a configured API client
+        api_client = ApiClientBuilder().build(CredentialsSource.secrets_path())
+
+        cls.corporate_actions_sources_api = lusid.CorporateActionSourcesApi(api_client)
+
+    def test_list_corporate_action_sources(self):
+        sources = self.corporate_actions_sources_api.list_corporate_action_sources()
+        for source in sources.values:
```

Comment:
Add an assertion here, e.g. number returned is > 0

Reply:
added self.assertGreater()